site_scons/soe_utils.py

```diff
@@ -28,7 +28,7 @@ def soelim(target, source, env):
 
 def soscan(node, env, path):
     c = node.get_text_contents()
-    return re.compile(r"^[\.']so\s+(\S+)", re.M).findall(c)
+    return re.compile(r"^[.']so\s+(\S+)", re.M).findall(c)
```

Comment:
you don't need to escape '.''s in a regex?

Reply:
not when they're inside certain other constructs - the brackets remove the need

Reply:
Confirmed...  Python docs describe this for sets `[]`:
>   Used to indicate a set of characters. In a set:
...
>        Special characters lose their special meaning inside sets. For example, `[(+\*)]` will match any of the literal characters '(', '+', '*', or ')'.

So characters which otherwise have special meaning are not escaped inside the brackets.